src/tests/jest.common.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
         'office-ui-fabric-react/lib/(.*)$': 'office-ui-fabric-react/lib-commonjs/$1',
         '@uifabric/utilities': '@uifabric/utilities/lib-commonjs',
         '@uifabric/styling': '@uifabric/styling/lib-commonjs',
+        '\\.(scss)$': 'identity-obj-proxy',
```

Comment:
This is unclear enough that it'd be nice to include a comment explaining it (maybe a link to an article explaining the technique?)

Reply:
Good catch! I'll add some info.